lib/gds_api/govuk_headers.rb

```diff
@@ -0,0 +1,14 @@
+module GdsApi
+  class GovukHeaders
+    class << self
+      def headers
+        Thread.current[:headers] ||= {}
+      end
+
+      def values
```

Comment:
I would expect this to just return the values side of the key/value equation, not the full hash. You know, like Python.